src/main/java/org/ibp/api/java/feedback/FeedbackService.java

```diff
@@ -8,4 +9,5 @@ public interface FeedbackService {
 
 	void dontShowAgain(FeedbackFeature feature);
 
+	FeedbackDto getFeedback(FeedbackFeature feature);
```

Comment:
quick comment @cuenyad you can return Optional also here and avppid handling null values